index.html

```diff
@@ -0,0 +1,55 @@
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width-device-width" initial-scale="1.0">
+        <title>My Calculator</title>
+        <link rel="stylesheet" type="text/css" href="style.css">        
+    </head>
+    <body>
+        <center>
+            <div class="body">
+                <h1>MY CALCULATOR</h1>
+                <table>

```

Comment:
Instead of using tables, for better responsiveness, you should checkout flexbox.